### PR TITLE
i18n(it): Translation /develop/index.mdx

### PR DESCRIPTION
**Pull request by loky-lp**
#### Description
i18n(it): Translation /develop/index.mdx
- What does this PR change?
  - It adds the index file for the "Develop" section, specifically the "Overview" page
- Closes # None.

Unfortunately I've started on the page before finding out about the [tracking website](https://v2.tauri.app/contribute/translate-status) and I've seen that this page it's not one that has been requested, so it's ok if this PR gets rejected, but since I've already done some work I think it will be helpful.

I’m looking forward to your valuable feedback and am excited at the prospect of contributing to tauri-docs. Thank you very much!

### CODE REVIEW

**Review by Graziano10 — APPROVED** [review]

**Comment by Teygeta** [comment]
I am Italian and I confirm that this work is really useful!

**Review by Teygeta — APPROVED** [review]

**Review by dreyfus92 — APPROVED** [review]
Awesome work peeps, thanks everyone! 😁

File: src/content/docs/it/develop/index.mdx [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+Se preferisci usare i DevTools del tuo browser, devi configurare [tauri-invoke-http](https://github.com/tauri-apps/tauri-invoke-http)
+per collegare le chiamate API di Tauri tramite un server HTTP.
+
+### Controllo del codice sorgente
+
+Nella repository del tuo progetto, **DOVRESTI** fare il commit del file `src-tauri/Cargo.lock` insieme al file `src-tauri/Cargo.toml` su git
+perché Cargo utilizza il file di lock per fornire compilazioni deterministiche.
+Inoltre, è raccomandato che tutte le applicazioni controllino il loro file `Cargo.lock`.
+**NON DOVRESTI** fare il commit della cartella `src-tauri/target` o qualsiasi suo contenuto.
```
Comment (dreyfus92) [inline src/content/docs/it/develop/index.mdx]:
```suggestion
**NON DOVRESTI** fare il commit della cartella `src-tauri/target` o qualsiasi suo contenuto.

```

File: src/content/docs/it/develop/index.mdx [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+title: Sviluppo
+description: Concetti fondamentali per lo sviluppo con Tauri.
+sidebar:
+  order: 0
+  label: Panoramica
+---
+
+import CommandTabs from '@components/CommandTabs.astro';
+
+Ora che hai [tutto configurato](../start/), sei pronto per avviare la tua applicazione con Tauri.
```
Comment (dreyfus92) [inline src/content/docs/it/develop/index.mdx]:
```suggestion
Ora che hai [tutto configurato](/start/), sei pronto per avviare la tua applicazione con Tauri.
```

File: src/content/docs/it/develop/index.mdx [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+
+Se stai usando un framework UI o un bundler di JavaScript, probabilmente avrai già accesso a un server
+locale che velocizza il processo di sviluppo, quindi, se non hai ancora configurato l'URL della tua app
+e lo script che lo fa partire, puoi farlo grazie ai campi [devUrl](../reference/config.md#devurl) e
```
Comment (dreyfus92) [inline src/content/docs/it/develop/index.mdx]:
```suggestion
e lo script che lo fa partire, puoi farlo grazie ai campi [devUrl](/reference/config.md#devurl) e
```

File: src/content/docs/it/develop/index.mdx [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+[beforeDevCommand](../reference/config.md#beforedevcommand) nel file di configurazione:
```
Comment (dreyfus92) [inline src/content/docs/it/develop/index.mdx]:
```suggestion
[beforeDevCommand](/reference/config.md#beforedevcommand) nel file di configurazione:
```

File: src/content/docs/it/develop/index.mdx [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+  label: Panoramica
+---
+
+import CommandTabs from '@components/CommandTabs.astro';
+
+Ora che hai [tutto configurato](/start/), sei pronto per avviare la tua applicazione con Tauri.
+
+Se stai usando un framework UI o un bundler di JavaScript, probabilmente avrai già accesso a un server
+locale che velocizza il processo di sviluppo, quindi, se non hai ancora configurato l'URL della tua app
+e lo script che lo fa partire, puoi farlo grazie ai campi [devUrl](/reference/config.md#devurl) e
```
Comment (dreyfus92) [inline src/content/docs/it/develop/index.mdx]:
```suggestion
e lo script che lo fa partire, puoi farlo grazie ai campi [devUrl](/reference/config#devurl) e
```

File: src/content/docs/it/develop/index.mdx [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+[beforeDevCommand](/reference/config.md#beforedevcommand) nel file di configurazione:
```
Comment (dreyfus92) [inline src/content/docs/it/develop/index.mdx]:
```suggestion
[beforeDevCommand](/reference/config#beforedevcommand) nel file di configurazione:
```

File: src/content/docs/it/develop/index.mdx [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+}
+```
+
+Nota che in questo esempio la cartella `src` deve contenere un file `index.html` insieme agli altri assets caricati dal tuo frontend.
+
+:::caution[Sicurezza del Server di Sviluppo Plain/Vanilla]
+
+Il server di sviluppo incorporato di Tauri non supporta l'autenticazione o la criptazione
+reciproca. Non dovresti mai usarlo per lo sviluppo su reti non affidabili.
+Leggi le [considerazioni sulla sicurezza del server di sviluppo](/security/lifecycle#development-server)
```
Comment (dreyfus92) [inline src/content/docs/it/develop/index.mdx]:
```suggestion
Leggi le [considerazioni sulla sicurezza del server di sviluppo](/security/lifecycle/#development-server)
```

File: src/content/docs/it/develop/index.mdx [hunk 8]
```diff
@@ -0,0 +1,249 @@
...
+---
+
+import CommandTabs from '@components/CommandTabs.astro';
+
+Ora che hai [tutto configurato](/start/), sei pronto per avviare la tua applicazione con Tauri.
+
+Se stai usando un framework UI o un bundler di JavaScript, probabilmente avrai già accesso a un server
+locale che velocizza il processo di sviluppo, quindi, se non hai ancora configurato l'URL della tua app
+e lo script che lo fa partire, puoi farlo grazie ai campi [devUrl](/reference/config#devurl) e
+[beforeDevCommand](/reference/config#beforedevcommand) nel file di configurazione:
```
Comment (dreyfus92) [inline src/content/docs/it/develop/index.mdx]:
```suggestion
[beforeDevCommand](/reference/config/#beforedevcommand) nel file di configurazione:
```

File: src/content/docs/it/develop/index.mdx [hunk 9]
```diff
@@ -0,0 +1,249 @@
...
+  label: Panoramica
+---
+
+import CommandTabs from '@components/CommandTabs.astro';
+
+Ora che hai [tutto configurato](/start/), sei pronto per avviare la tua applicazione con Tauri.
+
+Se stai usando un framework UI o un bundler di JavaScript, probabilmente avrai già accesso a un server
+locale che velocizza il processo di sviluppo, quindi, se non hai ancora configurato l'URL della tua app
+e lo script che lo fa partire, puoi farlo grazie ai campi [devUrl](/reference/config#devurl) e
```
Comment (dreyfus92) [inline src/content/docs/it/develop/index.mdx]:
```suggestion
e lo script che lo fa partire, puoi farlo grazie ai campi [devUrl](/reference/config/#devurl) e
```

**Comment by loky-lp** [comment]
@dreyfus92 Mega job 🎉
Can i ping you in the next translations?
I was waiting to get this PR merged before contributing again to let you evaluate my work.
If all the thing are ok I'm gonna translate the whole guide.

**Comment by dreyfus92** [comment]
Yes, feel free to ping me, although I always get pinged cause i'm part of the `wg-doc` group, and again thanks for taking the time in translating the docs 🫡

**Pull request merged**